scripts/scripts.js

```diff
@@ -32,7 +32,7 @@ import {
 const LCP_BLOCKS = []; // add your LCP blocks to the list
 
 // regex to find abstract paragraph
-export const ABSTRACT_REGEX = /(.*?);.*?(\d{4})|(.*?)(\d{4})\s+–\s+\b|(.*?)(\d{4})\s+-\s+\b/;
+export const ABSTRACT_REGEX = /(.*?);.*?(\d{4})|(.*?)(\d{4})\s+(–|-|‒)\s+|(.*?)(\d{4} (–|-|‒|:))(.)|(.*?)(\d{4}(–|-|‒|:)(.))/;
```

Comment:
@jjjaspher looks like you're supporting the following cases
- one space after the year and then any of `–|-|‒|:` after the year
- no spaces after the year and then any of `–|-|‒|:` after the year
I think we can combine that into the following condition
```
/(.*?);.*?(\d{4})|(.*?)(\d{4})\s*(–|-|‒|:)\s+/
```

Can you test if this matches all your requirements?

Reply:
@ravkiran, with that we can't capture these format
1. with space before (–|-|‒|:) and no space after (–|-|‒|:)  =  **"Lisboa, 28 de junho de 2020 -A Accenture Portugal e a"**
2. no before (–|-|‒|:) and no space after (–|-|‒|:)  =  **"Lisboa, 28 de junho de 2020-A Accenture Portugal e a"**

I haven't check all new articles if it's existing I just assumed there might be one. Is it fine to not consider this for now?

Reply:
@jjjaspher both cases you've mentioned is actually only one case which is `no space after (–|-|‒|:)`. With `\s*` in the regex, it matches 0 or more spaces after the year, but we've deliberately put `\s+` after the `(–|-|‒|:),` that's why it's not matching.
We can fix that by using
```
/(.*?);.*?(\d{4})|(.*?)(\d{4})\s*(–|-|‒|:)\s*/
```
But I think the space after `(–|-|‒|:)` is good to make it more readable, otherwise it looks odd, so I'd like to ignore this case and call that as fix in the authoring. But if you see more cases, we can change later.
So, if you're ok with this explanation, I'll commit
```
/(.*?);.*?(\d{4})|(.*?)(\d{4})\s*(–|-|‒|:)\s+/
```
in my pr and you can close this.
If you want to release the restriction of space after `(–|-|‒|:)`, I'll commit
```
/(.*?);.*?(\d{4})|(.*?)(\d{4})\s*(–|-|‒|:)\s+/
```
Please let me know.

Reply:
_"But I think the space after (–|-|‒|:) is good to make it more readable, otherwise it looks odd"_

@ravkiran I agree with this but that missed spacing by the author (might be in the future) has an impact the way being rendered on the newscard. I am thinking if we can capture already the **no space after (–|-|‒|:)** , just for the safer side? Can we go then with? **/(.*?);.*?(\d{4})|(.*?)(\d{4})\s*(–|-|‒|:)\s*/**

If it's fine with you, I'll close this PR and you can include it to your PR and merged it. Thank you